scripts/release-hooks.mjs

```
@@ -12,37 +13,30 @@ const root = new URL("..", import.meta.url).pathname.replace(/^\/([A-Z]:)/, "$1"
+  skip: SKIPPED_PACKAGES,
+});
 
-// Bump Version in manifest.json files. Discovered dynamically — the same way as
-// the package.json files above — rather than a hardcoded list: scan every
-// `packages/*/<plugin-folder>/manifest.json` that declares a `Version`. The old
-// static array silently skipped the Ulanzi plugin's manifest (it stayed at
-// 1.22.0.0 while the others advanced), exactly the drift issue #435 fixed for
-// package.json; auto-discovery picks up any current or future plugin manifest
-// (`.sdPlugin` / `.ulanziPlugin`) automatically.
+// Manifests are anchored to real plugin folders (`*.sdPlugin` / `*.ulanziPlugin`)
+// so an unrelated `packages/*​/<dir>/manifest.json` that happens to declare a
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)